core/storage/question/gae_models.py

```diff
@@ -566,7 +576,7 @@ def get_by_creator_id(cls, creator_id):
             creator_id: str. The user ID of the creator of the question.
 
         Returns:
-            QuestionSummaryModel. The summary model of the question.
+            list(QuestionSummaryModel). The summary model of the question.
```

Comment:
Thanks for the changing this, can you also change the description correctly?

Reply:
Is this resolved?

Reply:
Done.